28_FFTBloom/main.cpp

```diff
@@ -885,11 +890,12 @@ class FFTBloomApp final : public examples::SimpleWindowedApplication, public app
 			firstAxisFFTElementsPerInvocationLog2 = elementsPerInvocationLog2;
 			firstAxisFFTWorkgroupSizeLog2 = workgroupSizeLog2;
 			m_imageFirstAxisFFTWorkgroupSize = uint16_t(1) << workgroupSizeLog2;
+			m_imageSecondAxisFFTNumWorkgroups = uint32_t(1) << firstAxisFFTHalfLengthLog2;
 		}
 
 		// Second axis FFT might have different dimensions
 		{
-			auto [elementsPerInvocationLog2, workgroupSizeLog2] = workgroup::fft::optimalFFTParameters(m_device->getPhysicalDevice()->getLimits().maxOptimallyResidentWorkgroupInvocations, m_marginSrcDim.width);
+			auto [elementsPerInvocationLog2, workgroupSizeLog2] = workgroup::fft::optimalFFTParameters(m_device->getPhysicalDevice()->getLimits().maxOptimallyResidentWorkgroupInvocations, m_marginSrcDim.width, m_device->getPhysicalDevice()->getLimits().minSubgroupSize);
```

Comment:
https://github.com/Devsh-Graphics-Programming/Nabla-Examples-and-Tests/pull/171/files#r1923739241